timescaledb/how-to-guides/compression/about-compression.md

```diff
@@ -58,6 +58,33 @@ compressed.
 
 </procedure>
 
+For more information, see the API reference for [`ALTER TABLE
+(compression)`][alter-table-compression] and
+[`add_compression_policy`][add_compression_policy].
+
+## View current compression policy
+To view the compression policy that you've set, run:
+```sql
+SELECT * FROM timescaledb_information.jobs
+  WHERE proc_name='policy_compression';
+```
+
+For more information, see the API reference for [`timescaledb_information.jobs`][timescaledb_information-jobs].
+
+## Remove compression policy
+To remove a compression policy, run:
+```sql
+SELECT remove_compression_policy('<TABLE>');
+```
+
+For example, to remove the compression policy from a hypertable named `cpu`:
+```sql
+SELECT remove_compression_policy('cpu');
+```
```

Comment:
We have a lot of inconsistency across our documentation around how we provide "templates" of commands and then show how to use them. IMO, this is extra effort and the `'<TABLE>'` isn't used anywhere else (generally). We either need to stick to Postgres parameter type names (`REGCLASS`) or just show the second example because we clearly say "to remove... from a hypertable named..".

Reply:
This is actually something we're trying to standardize, see the [work in progress here](https://github.com/timescale/docs/pull/895). I agree this example is simple enough that it's unnecessary; I'll edit it out.